### PR TITLE
bio-containers annotation import

### PR DESCRIPTION
**Pull request by OlegZharkov**
As we discussed today with @hmenager @bgruening @hansioan @redmitry 
Here is the PR that parses [biocontainers annotations](https://raw.githubusercontent.com/BioContainers/tools-metadata/master/annotations.yaml). Currently CI usses this [branch](https://github.com/bio-tools/content-ecosystem-utils/tree/biocontainers-annotation-import) of `content-ecosystem-utils`
Current strategy:
 If we find a new tool, we create new directory under `data/`. (this is of course disputable. Any other ideas are always appreciated, please say so if you think otherwise)

Example of PR created by bot: https://github.com/OlegZharkov/content/pull/204

### CODE REVIEW

File: .github/workflows/biocontainers-import.yaml
Comment (bgruening) [inline .github/workflows/biocontainers-import.yaml:19]: biocontainers :)

Comment (OlegZharkov) [reply .github/workflows/biocontainers-import.yaml]: right, thanks!

**Comment by hmenager** [comment]
Thanks a lot for acting so quickly on that one @OlegZharkov. I have two comments:
- I cannot see any place in the content import script where the existing biocontainers YAML files get removed, before to do the import. Could you add this? My point is, similarly to the issue I recently had with bio.tools, we need to avoid keeping containers that have been removed in the ecosystem repo.
- also, let's not forget to merge the `biocontainers-annotation-import` branch to `master` in the `content-ecosystem-utils` repo

**Comment by OlegZharkov** [comment]
Hi @hmenager,


> * I cannot see any place in the content import script where the existing biocontainers YAML files get removed, before to do the import. Could you add this? My point is, similarly to the issue I recently had with bio.tools, we need to avoid keeping containers that have been removed in the ecosystem repo.

I implemented deleting old tools with https://github.com/bio-tools/content-ecosystem-utils/commit/2feca6a27ba7fa8196bfdf142dc851cae2aaeb92. It should do the job.

> * also, let's not forget to merge the `biocontainers-annotation-import` branch to `master` in the `content-ecosystem-utils` repo

No worries, I will merge `biocontainers-annotation-import` in `master` as soon as we get an approval here. I was just a bit hesitant to commit my stuff directly in `master`. Otherwise, I can actually merge it right now, if you don't mind.


@hansioan and @bgruening should still come to an agreement about new biocontainers tools, that are absent in out `content/data`. Do we create new directory for each new tool or do we put all new tools in one single directory? (hopefully, we'll talk about this during the next call)

**Comment by bgruening** [comment]
> @hansioan and @bgruening should still come to an agreement about new biocontainers tools, that are absent in out content/data. Do we create new directory for each new tool or do we put all new tools in one single directory? (hopefully, we'll talk about this during the next call)

I still think it should be a separate tool. If a tool does not have a bio.tool ID, its still a tool, can be in Galaxy ([we know about 106](https://github.com/galaxyproject/tools-iuc/blob/9b5f707621e8ad7cfd6977c22859bd19b5497803/missing_biotools.txt) already) and all have BioConda, BioContainers etc... Means all those tools can have a lot of metadata already, despite being not in bio.tools. At some point, we need to iterate over them and assign proper bio.tool IDs - but until this I think we should treat them like regular tools.

**Comment by hmenager** [comment]
Thanks a lot @OlegZharkov for the biocontainers files removal :)
Regarding the import of non-existent bio.tools files, I think it is important to have them. We should have them somewhere in this repository, but if we want to avoid synonym-confusion, we need to have them in separate folders at import stage.

**Review by hmenager — APPROVED** [review]

**Comment by OlegZharkov** [comment]
thanks @hmenager

**Pull request merged**
